### PR TITLE
logging for dev, logging to file and skipping noise

### PR DESCRIPTION
**Pull request by YashdalfTheGray**
Enabled logging for dev, all of them going to file obviously. Also only logs when hitting API endpoints in prod. 

Reviewers: @Mctalian, @davethenipper

### CODE REVIEW

**Comment by Mctalian** [comment]
Looks good to me, Approved.

File: server/server.js [hunk 1]
```diff
@@ -39,14 +41,24 @@ app.use(bodyParser.json());
...
         chalk.gray('":referrer" ":user-agent" ') +
-        'time=:response-time ms'
+        'time=:response-time ms',
+        {
+            stream: logStream,
+            skip: function(req) {
+                "use strict";
+                // TODO - Once the API change goes through, this will
+                // become return !/api\/v1\/|sse\//.test(req.originalUrl);
+                return !/ideas|users|events|comments/.test(req.originalUrl);
```
Comment (alanlgirard) [inline server/server.js]: Might as well change this line with PR #250 being approved

**Comment by alanlgirard** [comment]
Looks good. Merge #250 first then make the small endpoint change.

Approved

**Comment by Mctalian** [comment]
Agreed, merge #250 first

**Comment by YashdalfTheGray** [comment]
Done, merging now.

**Pull request merged**